cli/tools/fmt.rs

```diff
@@ -549,11 +551,7 @@ pub fn format_sql(
   // Add single new line to the end of file.
   formatted_str.push('\n');
 
-  Ok(if formatted_str == file_text {
-    None
-  } else {
-    Some(formatted_str)
-  })
+  Ok(Some(formatted_str))
```

Comment:
I think this should be reverted? Was there no test that fails?

Reply:
https://github.com/denoland/deno/pull/27014/files#diff-fe46255b8864b7c270c3f211fd444d65e0290b72bdf1bc5aff9c1198c3576ff2